rskj-core/src/main/java/co/rsk/trie/TrieStoreImpl.java

```diff
@@ -55,9 +60,17 @@ public TrieStoreImpl(KeyValueDataSource store) {
      */
     @Override
     public void save(Trie trie) {
+        noRetrieves = 0;
+        noSaves = 0;
+        noNoSaves = 0;
```

Comment:
I find the `no` prefix confusing, `noNoSaves` sounds like a double negation. Consider changing for `numberNoSaves`?

Reply:
prefixed those fields with `numOf`